apps/client/ui/MainPanel/MainPanel.tsx

```diff
@@ -54,10 +85,21 @@ const MainPanel: FC = () => {
         </div>
       </header>
       <section>
-        <MoviesList className="py-5" />
-        <Button variant="secondary" className="mt-9">
-          Load more
-        </Button>
+        {isLoading && 'Loading...'}
```

Comment:
great you prepared UI for states of list 💪 
you might do it a little simpler (in a logic way) you can create some `renderList` function above and there just use `if` statements. it's simpler this way that you can place importance of states in order and no need to watch the statuses for less important states (like `!isLoading && !error` here) 👌 you can also add status for empty list if `loaded && length === 0` (not necessary if you think it's overkill) 😄  it would looks like this:
```
if (isLoading) return <loading ui>
if (error) return <error ui>
if (isLoaded && list.length === 0) return <empty ui>
return <list>
```